lib/gui/app/components/drive-selector/DriveSelectorModal.jsx

```diff
@@ -0,0 +1,320 @@
+/*
+ * Copyright 2019 balena.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict'
+
+const _ = require('lodash')
+const React = require('react')
+const { Modal } = require('rendition')
+const {
+  isDriveValid,
+  getDriveImageCompatibilityStatuses,
+  hasListDriveImageCompatibilityStatus,
+  COMPATIBILITY_STATUS_TYPES
+} = require('../../../../shared/drive-constraints')
+const store = require('../../models/store')
+const analytics = require('../../modules/analytics')
+const availableDrives = require('../../models/available-drives')
+const selectionState = require('../../models/selection-state')
+const { bytesToClosestUnit } = require('../../../../shared/units')
+const utils = require('../../../../shared/utils')
+const { open: openExternal } = require('../../os/open-external/services/open-external')
+const ConfirmModal = require('../confirm-modal/ConfirmModal.jsx')
+
+const DriveSelectorModal = (props) => {
```

Comment:
```suggestion
const DriveSelectorModal = ({close}) => {
```
And remove the line below.